backend/.env.example

```diff
@@ -2,8 +2,8 @@
 MODEL="azureai"
 MISTRAL_LARGE_URL=MISTRAL_LARGE_ENDPOINT
 MISTRAL_LARGE_KEY=MISTRAL_LARGE_API_KEY
-NEO4j_URI="bolt://localhost:7687"
```

Comment:
`localhost` won't work when the backend is running in a Docker container but neo4j is running outside Docker.

To make a request from a container to the local host, we must use `host.docker.internal`. 

We need:
```
NEO4J_URI="bolt://host.docker.internal:7687"
```

Reply:
@Ch0live - how to you want to handle this?

I think we either change `NEO4J_URI` in the `.env.example` or we include a comment in the `README` on how to configure the `NEO4J_URI` when the backend is running within Docker (and neo4j is not).

Reply:
Still getting an error
```
Error: {code: Neo.ClientError.Security.Unauthorized} {message: The client is unauthorized due to authentication failure.}
```
Unsure on the fix for this :/

Reply:
Fixed! It was a permission issue due to an incorrect password